stage1/init/init.go

```diff
@@ -217,10 +203,24 @@ func getArgsEnv(p *Pod, flavor, systemdVersion string, debug bool) ([]string, []
 		} else {
 			args = append(args, fmt.Sprintf("--register=false"))
 		}
+
 	default:
 		return nil, nil, fmt.Errorf("unrecognized stage1 flavor: %q", flavor)
 	}
 
+	if util.IsRunningSystemd() && systemdVersion > 215 {
```

Comment:
// Check both if systemd is running on the host and the version of systemd in stage1.
function isJournalSupportedInStage1
return is version in {  "v219", "v220", "master"}